packages/storybook/src/stories/Accordion.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+export default {
+  title: 'Design System/Accordion',
+  component: Accordion,
+} as ComponentMeta<typeof Accordion>
+
+const Template: ComponentStory<typeof Accordion> = (args) => <Accordion {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  children: (
+    <AccordionItem>
+      <AccordionSummary>Heading</AccordionSummary>
+      <AccordionDetails>Content</AccordionDetails>
+    </AccordionItem>
+  ),
+}
+
+export const SingleExpanded = Template.bind({})
+SingleExpanded.args = {
+  children: (
+    <AccordionItem>
+      <AccordionSummary isExpanded={true}>Heading</AccordionSummary>
+      <AccordionDetails isExpanded={true}>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+        labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+        cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
+      </AccordionDetails>
+    </AccordionItem>
+  ),
+}
+
+const multileStoryData = [
+  {
+    summary: 'Heading1',
+    details: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+    incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+    exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+    dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+    Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+    mollit anim id est laborum`,
+  },
+  {
+    summary: 'Heading2',
+    details: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+    incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+    exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+    dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+    Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+    mollit anim id est laborum`,
+  },
+  {
+    summary: 'Heading3',
+    details: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+    incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+    exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+    dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+    Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+    mollit anim id est laborum`,
+  },
+]
+const MultipleStory = () => {
+  const { activeItem, handleItemClick } = useAccordionController()
```

Comment:
Optional: Now that I see it, it might be better to let the client name these two variables as they see fit. So instead of returning an object, maybe we return an array.

`const [activeItem, handleItemClick] = useAccordionController()`

What do you think?

Reply:
I think Object is easier and better to use rather than an array.